modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Image.js

```diff
@@ -69,6 +102,14 @@ export default options => {
       //     alt="Elephant at sunset">
       //   <figcaption>An elephant at sunset</figcaption>
       // </figure>
+      // OR with link
+      // <figure>
+      //   <a href="https://example.com">
+      //     <img src="/media/cc0-images/elephant-660-480.jpg"
+      //       alt="Elephant at sunset">
+      //   </a>
+      //   <figcaption>An elephant at sunset</figcaption>
+      // </figure>
```

Comment:
Remove?

Reply:
I prefer to leave it for future reference (showcases the generated markup). Added a dev comment